src/nls/locale-de.json

```diff
@@ -116,6 +124,58 @@
     },
     "offlineMsg" : "Du scheinst keine Verbindung zum Internet zu haben. Die Karte funktioniert nur, wenn du online bist. Du kannst deinen Standort manuell setzen oder dies später tun."
   },
+  "wizard-upload-configfile": {
+    "cert": {
+      "label": "VPN Zertifikat",
+      "select": "Zertifikatsdatei",
+      "error": {
+        "pattern": "Falsches Dateiformat oder falscher Dateiname. Nur Textdateien sind erlaubt."
+      }
+    },
+    "key": {
+      "label": "VPN Schlüssel",
+      "select": "Schlüsseldatei",
+      "error": {
+        "pattern": "Falsches Dateiformat oder falscher Dateiname. Nur Textdateien sind erlaubt."
+      }
+    },
+    "takey": {
+      "label": "TLS Auth Schlüssel",
+      "select": "Schlüsseldatei",
+      "error": {
+        "pattern": "Falsches Dateiformat oder falscher Dateiname. Nur Textdateien sind erlaubt."
+      }
+    },
+    "cacert": {
+      "label": "VPN CA Zertifikat",
+      "select": "Zertifikatsdatei",
+      "error": {
+        "pattern": "Falsches Dateiformat oder falscher Dateiname. Nur Textdateien sind erlaubt."
+      }
+    },
+    "secret": {
+      "label": "VPN secret",
```

Comment:
we should remove the prefix "VPN" in all labels. It's clear from the parent checkbox that this is for a VPN.